asmcli/commands/help.sh

```diff
@@ -62,11 +62,14 @@ OPTIONS:
   -l|--cluster_location  <LOCATION>   The GCP location of the target cluster.
   -n|--cluster_name      <NAME>       The name of the target cluster.
   -p|--project_id        <ID>         The GCP project ID.
-  --kc|--kubeconfig      <KUBECONFIG> Path to the kubeconfig file to use for CLI requests.
+  --kc|--kubeconfig <KUBECONFIG_FILE> Path to the kubeconfig file to use for CLI requests.
                                       Required if not supplying --cluster_location,
                                       --cluster_name, --project_id in order to locate
                                       and connect to the intended cluster.
   --ctx|--context        <CONTEXT>    The name of the kubeconfig context to use.
+  --fleet_id             <FLEET ID>   The ASM Fleet host project ID. Required for non-GCP
```

Comment:
Let's avoid the phrasing "ASM Fleet". In this instance I think we can just leave out the word "ASM" entirely and say "The Fleet host project ID."

If you know what at Fleet is it makes sense, if you don't then adding ASM isn't going to help.

Reply:
done